data.sql

```diff
@@ -5,4 +5,11 @@ VALUES
 ('Gabumon','2018-11-15',2,8,true), 
 ('Pikachu','2021-01-07',1,15.04,false),
 ('devimon','2017-05-12',5,11,true),
+('Charmander','2020-03-08',0,11,false),
+('Plantmon' , '2021-11-15',2,5.7,true),
+('Squirtle','1993-04-02',3,12.13,false),
```

Comment:
your data are incorrect please check them, correct them properly and check again for a re-review

Reply:
Not fullfield.